apps/omg_watcher/test/omg_watcher/exit_processor/core_test.exs

```diff
@@ -1356,8 +1460,8 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
         |> Core.new_piggybacks([%{tx_hash: tx_hash, output_index: 1}])
 
       finalization = %{in_flight_exit_id: ife_id, output_index: 1}
-      {:ok, processor, _} = Core.finalize_in_flight_exits(processor, [finalization])
-      {:ok, ^processor, []} = Core.finalize_in_flight_exits(processor, [finalization])
+      {:ok, processor, _} = Core.finalize_in_flight_exits(processor, [finalization], %{})
+      {:ok, ^processor, _} = Core.finalize_in_flight_exits(processor, [finalization], %{})
```

Comment:
here I think the assertion that no more stuff is written do db was relevant. Pls revert the `[]`

Reply:
This was done on purpose. Those db_updates returned were in fact current state of ifes after finalizations are applied to the core. Even if nothing changes, db_updates will be produced. There should be a check that those updates are the same as in the line above.

After giving it a second thought, it feels more nautral to expect (as you expected) that those updates are "real" updates and are only produced when ifes are in fact changing. I refactored that.